src/app/views/App.tsx

```diff
@@ -460,32 +425,16 @@ class App extends Component<IAppProps, IAppState> {
                 handleClasses={{
                   right: classes.vResizeHandle
                 }}
-                bounds={'window'}
+                bounds={'parent'}
                 size={{
                   width: sideWidth,
                   height: ''
                 }}
               >
-
-                {mobileScreen && appTitleDisplayOnMobileScreen(
-                  stackTokens,
-                  classes,
-                  this.toggleSidebar
-                )}
-
-                {!mobileScreen && appTitleDisplayOnFullScreen(
-                  classes,
-                  minimised,
-                  this.toggleSidebar
-                )}
-
-                <hr className={classes.separator} />
-
-                {this.displayAuthenticationSection(minimised)}
-                <hr className={classes.separator} />
-
-                {showSidebar && ( <Sidebar currentTab = { this.state.sidebarTabSelection }
-                  setSidebarTabSelection = { this.setSidebarTabSelection } /> ) }
+                <Sidebar currentTab = { this.state.sidebarTabSelection }
```

Comment:
*Question:* Why have we passed the visibility as a prop down to the child instead of hiding it from the parent?